vignettes/dataset-status.Rmd

```diff
@@ -0,0 +1,37 @@
+---
+title: "dataset-status"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{dataset-status}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Dataset status is recorded below. Please note that due to our release schedule datasets may remain non-functional if broken using the CRAN version for some time even if fixed on GitHub. Also note that transcient issues may affect our testing of datasets and so our checks may indicate occassionally show a spurious failure. 
```

Comment:
What do we mean by "*working*" in the list below for CRAN status? Do we plan to have a workflow which downloads the latest CRAN version and tests that for comparison?

Reply:
So essentially what I planned here was to update datasets as they are detected to break by our dataset checks. This will mean they only are initially fixed on GitHub and hence needing to add a separate tag indicating an issue with the CRAN version. When we update on CRAN can then reset back to working (assuming we have fixed the problems). There might be a smarter way of handling this but I am not sure what it is.